includes/data-stores/class-wc-reports-orders-data-store.php

```diff
@@ -95,18 +181,22 @@ public function get_data( $query_args ) {
 		global $wpdb;
 
 		$table_name = $wpdb->prefix . self::TABLE_NAME;
-		$now       = time();
-		$week_back = $now - WEEK_IN_SECONDS;
-
-		$defaults = array(
-			'per_page' => get_option( 'posts_per_page' ),
-			'page'     => 1,
-			'order'    => 'DESC',
-			'orderby'  => 'date',
-			'before'   => date( WC_Reports_Interval::$iso_datetime_format, $now ),
-			'after'    => date( WC_Reports_Interval::$iso_datetime_format, $week_back ),
-			'interval' => 'week',
-			'fields'   => '*',
+		$now        = time();
+		$week_back  = $now - WEEK_IN_SECONDS;
+
+		// These defaults are only applied when not using REST API, as the API has its own defaults that overwrite these.
+		$defaults   = array(
+			'per_page'     => get_option( 'posts_per_page' ),
+			'page'         => 1,
+			'order'        => 'DESC',
+			'orderby'      => 'date',
+			'before'       => date( WC_Reports_Interval::$iso_datetime_format, $now ),
+			'after'        => date( WC_Reports_Interval::$iso_datetime_format, $week_back ),
+			'interval'     => 'week',
+			'fields'       => '*',
+			'categories'   => array(),
+			'coupons'      => array(),
+			'order_status' => 'completed',
```

Comment:
This should use `self::get_report_order_statuses()` instead of just `'completed'`.

Reply:
Makes sense, updated.